src/test/scala/io/confluent/examples/streams/GenericAvroScalaIntegrationTest.scala

```diff
@@ -66,42 +69,28 @@ class GenericAvroScalaIntegrationTest extends AssertionsForJUnit {
     //
     // Step 1: Configure and start the processor topology.
     //
-    val builder: StreamsBuilder = new StreamsBuilder()
+    val builder = new StreamsBuilder
 
     val streamsConfiguration: Properties = {
       val p = new Properties()
       p.put(StreamsConfig.APPLICATION_ID_CONFIG, "generic-avro-scala-integration-test")
       p.put(StreamsConfig.BOOTSTRAP_SERVERS_CONFIG, cluster.bootstrapServers())
-      p.put(StreamsConfig.DEFAULT_KEY_SERDE_CLASS_CONFIG, Serdes.ByteArray.getClass.getName)
-      p.put(StreamsConfig.DEFAULT_VALUE_SERDE_CLASS_CONFIG, classOf[GenericAvroSerde])
       p.put(AbstractKafkaAvroSerDeConfig.SCHEMA_REGISTRY_URL_CONFIG, cluster.schemaRegistryUrl)
       p.put(ConsumerConfig.AUTO_OFFSET_RESET_CONFIG, "earliest")
       p
     }
 
-    // Write the input data as-is to the output topic.
-    //
-    // Normally, because a) we have already configured the correct default serdes for keys and
-    // values and b) the types for keys and values are the same for both the input topic and the
-    // output topic, we would only need to define:
-    //
-    //   builder.stream(inputTopic).to(outputTopic);
-    //
-    // However, in the code below we intentionally override the default serdes in `to()` to
-    // demonstrate how you can construct and configure a generic Avro serde manually.
-    val stringSerde: Serde[String] = Serdes.String
-    val genericAvroSerde: Serde[GenericRecord] = {
+    // Make an implicit serde available for GenericRecord, which is required for operations such as `to()` below.
+    implicit val genericAvroSerde: Serde[GenericRecord] = {
       val gas = new GenericAvroSerde
-      // Note how we must manually call `configure()` on this serde to configure the schema registry
```

Comment:
Why do we remove this comment?

Reply:
I found the comment unnecessary because we are not setting default serde options any longer (thanks to the Scala API's "magic" around serdes), so the comment doesn't make a whole lot of sense.

And the fact that we do explicitly `configure()` the serde should be enough to convey that "this is required" to the user.